src/System.Windows.Forms/src/System/Windows/Forms/TaskDialogProgressBar.cs

```diff
@@ -49,21 +49,24 @@ public TaskDialogProgressBar(TaskDialogProgressBarState state)
         ///   where the default value is <see cref="TaskDialogProgressBarState.None"/>.
         /// </value>
         /// <remarks>
-        /// <para>
-        ///   This control will only be shown if this property is not
-        ///   <see cref="TaskDialogProgressBarState.None"/>.
-        /// </para>
-        /// <para>
-        ///   This property can be set while the dialog is shown. However, while the dialog is
-        ///   shown, it is not possible to change the state from
-        ///   <see cref="TaskDialogProgressBarState.None"/> to any other state,
-        ///   and vice versa.
-        /// </para>
+        ///   <para>
+        ///     This control will only be shown if this property is not
+        ///     <see cref="TaskDialogProgressBarState.None"/>.
+        ///   </para>
+        ///   <para>
+        ///     This property can be set while the dialog is shown. However, while the dialog is
+        ///     shown, it is not possible to change the state from
+        ///     <see cref="TaskDialogProgressBarState.None"/> to any other state,
+        ///     and vice versa.
+        ///   </para>
         /// </remarks>
+        /// <exception cref="InvalidOperationException">
+        ///   Attempt to hide the progress bar by setting the value 
```

Comment:
This doesn't read quite right to me. Maybe:

```suggestion
        ///   The value is <c>TaskDialogProgressBarState.None</c> and the dialog is displayed.
```

Reply:
Updated, thank you